Package.swift

```diff
@@ -78,6 +78,12 @@ let package = Package(
                 "Tests/ObjCTokenParserTests.m"
             ],
             sources: ["Tests"]
+        ),
+        .testTarget(
+            name: "AzureCommunicationChatTests",
+            dependencies: ["AzureCommunication", "AzureCommunicationChat", "OHHTTPStubsSwift"],
```

Comment:
Will remove OHHTTPStubs for now - I was using this to support record/playback mode, this PR just supports live mode for now.